plugins/package-managers/gradle-model/src/main/kotlin/Extensions.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
```

Comment:
Do you want to updated the year?

Reply:
I deliberately took the year in which the original functions were introduced.